File: P0/back/models/event.py
from .constanst import EventType
from pydantic import BaseModel
from datetime import datetime

class Event(BaseModel):
    id: str
    name: str
    eventType: EventType
    place: str
    address: str
    startDate: datetime
    endDate: datetime
    isVirtual: bool
    owner_id: str

    class Config:
        orm_mode = True

File: P0/front/src/models/event.js
class Event {
  constructor(
    id,
    name,
    category,
    place,
    address,
    startDate,
    endDate,
    isVirtual,
    owner,
    ownerId
  ) {
    this.id = id;
    this.name = name;
    this.category = category;
    this.place = place;
    this.address = address;
    this.startDate = startDate;
    this.endDate = endDate;
    this.isVirtual = isVirtual;
    this.owner = owner;
    this.ownerId = ownerId;
  }
}

export { Event };
